packages/discord.js/src/client/websocket/handlers/index.js

```diff
@@ -53,6 +53,7 @@ const handlers = Object.fromEntries([
   ['GUILD_SCHEDULED_EVENT_DELETE', require('./GUILD_SCHEDULED_EVENT_DELETE')],
   ['GUILD_SCHEDULED_EVENT_USER_ADD', require('./GUILD_SCHEDULED_EVENT_USER_ADD')],
   ['GUILD_SCHEDULED_EVENT_USER_REMOVE', require('./GUILD_SCHEDULED_EVENT_USER_REMOVE')],
+  ['APPLICATION_COMMAND_PERMISSIONS_UPDATE', require('./APPLICATION_COMMAND_PERMISSIONS_UPDATE')],
```

Comment:
If you run the [generator script](https://github.com/discordjs/discord.js/blob/main/packages/discord.js/scripts/generateRequires.mjs), this will be moved to the top.